decidim-proposals/spec/commands/decidim/comments/create_comment_with_proposal_links_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Comments
+    describe CreateComment do
+      include_context "when creating a comment"
+      it_behaves_like "create comment"
+
+      context "and comment contains a link to a proposal" do
```

Comment:
Can you add a test checking it works when adding a full link? I see it's tested in the ContentParser, but I think it'd be good if we double-check this from the user point of view 😄